test/unit/sourcefile_tests.py

```diff
@@ -29,7 +29,7 @@ def test_sha1_file_encoding(self):
         self.assertEqual("ac7180f6b038d5ae4f2297989e39a900995bb8fc", sha1_key)
     
     def test_make_oembed_url(self):
-        v_urls = ['https://vimeo.com/7100569', 'https://www.youtube.com/watch?v=bDOYN-6gdRE']
+        v_urls = ['https://vimeo.com/7100569', 'https://www.youtube.com/watch?v=bDOYN-6gdRE&si=123']
```

Comment:
Let's keep the URL without the `si` query string, and add one with it, both at the start of the query string, and perhaps one the the middle of two query parameters.